src/components/NavigationBar.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import BottomNavigation from '@mui/material/BottomNavigation';
+import BottomNavigationAction from '@mui/material/BottomNavigationAction';
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import GroupsRoundedIcon from '@mui/icons-material/Groups';
+import HomeRoundedIcon from '@mui/icons-material/HomeRounded';
+import NotificationsOutlinedIcon from '@mui/icons-material/NotificationsOutlined';
+import PersonOutlineOutlinedIcon from '@mui/icons-material/PersonOutlineOutlined';
+import "../css/HomePage.css"
+
+
+export default function SimpleBottomNavigation() {
+  const [value, setValue] = React.useState(0);
```

Comment:
remove unused `value`

Reply:
Need this value to select the icon which is active and display its css properties